frontEnd/tickets-app/src/components/Main/Main.js

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import SignIn from "../../pages/SignIn/SignIn";
+import { Route, Routes } from "react-router-dom";
+import Footer from "../Footer/Footer";
+import { useSelector } from "react-redux";
+import { USER } from "../../store/user/selectors/selectors";
```

Comment:
selectors are simple function, so you should call them like `getUser`, or `selectUser`